pom.xml

```diff
@@ -150,7 +150,7 @@
     <version.org.apache.neethi>3.1.1</version.org.apache.neethi>
     <version.org.apache.poi>4.1.2</version.org.apache.poi>
     <version.org.apache.sshd>2.9.2</version.org.apache.sshd>
-    <version.org.apache.tomcat>6.0.32</version.org.apache.tomcat>
+    <version.org.apache.tomcat>9.0.84</version.org.apache.tomcat>
```

Comment:
this is probably good to have latest, but i was also looking into matching the JWS which now uses 9.0.62, but i think we can try to use the latest, greatest ;)

Reply:
the only problem now is we use/declare another version here https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/2413/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R280